src/session/src/context.rs

```diff
@@ -38,6 +38,8 @@ pub type ConnInfoRef = Arc<ConnInfo>;
 
 const CURSOR_COUNT_WARNING_LIMIT: usize = 10;
 
+pub const ANALYZE_FORMAT_HEADER_NAME: &str = "x-greptime-analyze-format";
```

Comment:
What about adding `query` to the header name? It's more precise.
```suggestion
pub const ANALYZE_FORMAT_HEADER_NAME: &str = "x-greptime-query-analyze-format";
```

Reply:
Isn't `explain-analyze-format`?

Reply:
I'll pick `x-greptime-query-analyze-format`, `explain analyze` is a verbose grammar thing.